recipes/podman/meta.yaml

```diff
@@ -0,0 +1,101 @@
+{% set version = "2.1.1" %}
+
+package:
+  name: podman
+  version: {{ version }}
+
+source:
+  url: https://github.com/containers/podman/archive/v{{ version }}.tar.gz
+  sha256: 5ebaa6e0dbd7fd1863f70d2bc71dc8a94e195c3339c17e3cac4560c9ec5747f8
+  folder: src
+  patches:
+    # NOTE: This adds the memfd_create syscall which is not available on CentOS 6's kernels.
+    - 0001-Add-missing-defines-for-CentOS-6.patch
+    - 0002-Use-CPP-CC-and-flags-in-dep-check-scripts.patch
+    # NOTE: If the patch for relative paths stops working, see the buildah
+    #       feedstock for an alternative with an hardcoded prefix path.
+    - 0003-Allow-relative-paths-for-system-config-files.patch
```

Comment:
I intend to
- not upstream `0001-Add-missing-defines-for-CentOS-6.patch`
  IMO, we (conda-forge) should update to cos7 in the foreseeable future.
  And I don't want to burden upstream with temporarily adding these patches.
- upstream `0002-Use-CPP-CC-and-flags-in-dep-check-scripts.patch`.
- work with upstream on how we can support the Conda use case with something less ugly than
  `0003-Allow-relative-paths-for-system-config-files.patch`.
  (Upstream was very forthcoming when @epruesse packaged `skopeo`, hence I believe we can work this out.)
  The patch works and I intend to maintain it until we have upstream support, but it's quite hacky and ideally we'd
  have something more robust that doesn't involve `FallbackToPathRelativeToExe` calls in random places etc,